scripts/scil_correlate_qc_dwi.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+"""
+
+import argparse
+
+from dipy.io.gradients import read_bvals_bvecs
+import nibabel as nib
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             add_force_b0_arg)
+from scilpy.utils.bvec_bval_tools import check_b0_threshold, normalize_bvecs
+import math
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_dwi',
+                   help='The DWI file (.nii) to concatenate.')
+    p.add_argument('in_bval',
+                   help='The b-values files in FSL format (.bval).')
+    p.add_argument('in_bvec',
+                   help='The b-vectors files in FSL format (.bvec).')
```

Comment:
Would a --mask option help?

Reply:
It could, but this aims to be a very first step.
Like flagging subjects to discard or look into before launching anything immediately.

It could work after the preprocessing entirely, in this case, the DWI is skull-strip anyway.